README.md

```diff
@@ -1,62 +1,66 @@
+Have an idea for a new feature? You're welcome to leave suggestions and ideas [here](https://neo4j-browser.canny.io/feature-requests).
 
-`yarn dev` to have continuous testing on every file change.
+Contributions welcome! More information in our [CONTRIBUTING.md](CONTRIBUTING.md).
 
-#### E2E Suite
+## Development 
+Running Neo4j Browser locally requires Node.js (^12.4.0) and for dependencies we use yarn (`npm install -g yarn`). 
+To install dependencies and then start the development server at `http://localhost:8080`:
 
-`yarn e2e` to run the cypress js test suite (requires a **fresh** installation of neo4j to run against, expects neo4j 3.5 by default).
-`yarn e2e --env server=3.3` to only run cypress js tests valid for neo4j server version 3.3.
+```shell
+yarn install
+yarn start
+```
+
+### Testing overview
+Neo4j Browser has both unit and end to end tests running automatically on every pull request. To run the tests locally:
+
+`yarn test-unit` runs a linter and then our unit tests.
+
+`yarn test-e2e` runs our Cypress end to end tests in the easiest, slowest way. Running them with this command requires docker installed and that nothing else runs on ports 7687 and 8080.
+
+#### Cypress e2e test suite in depth
 
-To run on an existing server (with a password already set), you can use any of these (the default password is set to "newpassword", pass in `--env browser-password=your-password`):  
-`yarn e2e-local --env server=3.4`  
-`yarn e2e-local-open --env server=3.4`  
-The latter just opens Cypress runner so you can see the tests being executed and run only some of them. Very useful when writing tests.
+`yarn e2e-open` to open the Cypress test runner (requires a **fresh** installation of Neo4j to run against, expects neo4j 3.5 by default). See details below on how to configure database version.
 
-There are also e2e tests that cover import from CSV files. To run these, copy the `e2e_tests/files/import.csv` to the `import/` directory of the database you want to run the tests on and then start the e2e tests with the `--env include-import-tests=true` flag.
-Example: `yarn e2e-local-open --env server=3.4,include-import-tests=true`
+`yarn e2e-local-open` to run against an existing server (with a password already set). We use `newpassword` as the default password here, make sure to pass your password:
+`yarn e2e-local-open --env browser-password=<your-password-here>`
 
-Here are the available options / env variables:
+To avoid opening the Cypress test runner and just run the tests in the terminal, remove the "-open" suffix from the previous two commands (so `yarn e2e` and `yarn e2e-local` respectively).
 
+So to run tests on your existing 4.2 database with the password "hunter2" without opening the Cypress visual test runner:
+`yarn e2e-local --env browser-password=hunter2,server=4.2`
+
+All the available options for `--env` are:
 ```
-server=3.2|3.3|3.4|3.5|4.0|4.1 (default 3.5)
+server=3.5|4.0|4.1|4.2 (default 3.5)
```

Comment:
Can we change the default to be 4.3 (and also add 4.3 to the list)?

Reply:
Updated, let's see if the tests still pass